bindings/pyroot_experimental/pythonizations/python/ROOT/_facade.py

```diff
@@ -179,3 +179,42 @@ def _setattr(self, name, val):
     @property
     def __version__(self):
         return self.gROOT.GetVersion()
+
+    # Overload VecOps namespace
+    @property
+    def VecOps(self):
+        if not 'VecOps' in self.__dict__:
+            ns = self._fallback_getattr('VecOps')
+            self.__dict__['VecOps'] = ns
+            try:
+                from libROOTPythonizations import AsRVec
+                ns.AsRVec = AsRVec
+            except:
+                pass
```

Comment:
We should perhaps error out here, since we were unable to pythonize the namespace (the same comment applies for `RDF` and `TMVA`).

Reply:
Can it happen that we have a namspace, like `RDF` but not the pythonization? In case yes, i would fail silently here and just give a "attribute not found" error by Python. Otherwise, we fail hard on any operation of the namespace.

Reply:
So we really have to be sure this is the correct thing to do.